cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=63, required=True)
+    description = serializers.CharField()
```

Comment:
since description can be null in your models, I suggest adding required=False
```suggestion
    description = serializers.CharField(required=False)
```

Reply:
Ah, thank you I didn't understand the documentation that way